doc/source/whatsnew/v0.21.0.txt

```diff
@@ -76,6 +76,7 @@ Removal of prior version deprecations/changes
 
 - ``pd.read_excel()`` has dropped the ``has_index_names`` parameter (:issue:`10967`)
 - ``Categorical`` has dropped the ``.order()`` and ``.sort()`` methods in favor of ``.sort_values()`` (:issue:`12882`)
+- ``pd.eval`` and ``DataFrame.eval`` have changed the default of ``inplace`` from ``None`` to ``False`` (:issue:`11149`)
```

Comment:
I like to use `` :func:`eval` `` so its a link (essentially we should always do this if feasible, maybe even make a mini-doc section on how to do this. e.g.  (OT). I think the `` :class:`~pandas.HDFStore` `` links are not working :<)